tests/console/lsof.pm

```diff
@@ -21,11 +21,13 @@ use strict;
 use testapi;
 use warnings;
 use utils 'zypper_call';
+use version_utils 'is_jeos';
 
 sub run {
     my ($self) = @_;
     select_console('root-console');
     zypper_call('in netcat');
```

Comment:
IMO those calls should be combined and be unconditional.

Reply:
I usually do that in case the packages are expected to be there by default.
Updated.
New validation run: http://ccret.suse.cz/tests/3196#step/lsof/1

Reply:
Right - if the test's purpose is to ensure lsof is already available. AFAICT that's not the case here and IMO it doesn't make much sense for lsof to be a hard requirement anyway.

Reply:
I updated this. Is there something else to be done ?

Reply:
It's looking good to me - but I don't have a merge button...